### PR TITLE
add defer toolbar build option

### PR DESCRIPTION
**Pull request by kand**

### CODE REVIEW

**Comment by kand** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;jWvOumfO&#x2F;40-3-rich-text"> (3) rich text</a>

**Comment by kand** [comment]
@collin @daytonn hack for multiple toolbars on a page

File: build/onion-editor.js [hunk 1]
```diff
@@ -5182,6 +5180,7 @@ define('scribe-plugin-curly-quotes',[],function () {
 
       // Apply a function on all text nodes in a container, mutating in place
       function mapTextNodes(container, func) {
+console.log(container)
         var walker = document.createTreeWalker(container, NodeFilter.SHOW_TEXT);
```
Comment (daytonn) [inline build/onion-editor.js]: Do we want to leave this console statement?

File: build/onion-editor.js [hunk 2]
```diff
@@ -11621,12 +11621,21 @@ define('onion-editor',[
     scribe.use(scribePluginCurlyQuotes());
     scribe.use(scribePluginKeyboardShortcuts(Object.freeze(keyCommands)));
 
-    //TODO: kill this existing toolbar & replace w/ Medium style selection toolbar
+    // garbage toolbar setup
```
Comment (daytonn) [inline build/onion-editor.js]: This comment does not document anything but your feelings on the setup, please remove it.

File: src/js/onion-editor.js [hunk 3]
```diff
@@ -341,12 +341,21 @@ define('onion-editor',[
     scribe.use(scribePluginCurlyQuotes());
     scribe.use(scribePluginKeyboardShortcuts(Object.freeze(keyCommands)));
 
-    //TODO: kill this existing toolbar & replace w/ Medium style selection toolbar
+    // garbage toolbar setup
```
Comment (daytonn) [inline src/js/onion-editor.js]: Same as above

**Comment by daytonn** [comment]
👍 other comments withstanding

**Pull request merged**